src/export/ExportWavPack.cpp

```diff
@@ -0,0 +1,431 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+
+   ExportWavPack.cpp
+
+   Subhradeep Chakraborty
+
+   This program is distributed under the GNU General Public License, version 2.
```

Comment:
Please update to GPL2-or-later.

Or even better, please add `/*  SPDX-License-Identifier: GPL-2.0-or-later */` to the top of both Export and Import

Reply:
done